components/automate-backend-elasticsidecar/habitat/automate-backend-elasticsidecar/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Contributor Covenant Code of Conduct
```

Comment:
There is no need of this file. Automate repo root already has this file

Reply:
ok removing it